src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -102,24 +108,31 @@ public Lift() {
     }
 
     protected double returnPIDInput() {
-        return liftEncoder.getDistance();
+        return liftHeight();
     }
 
     protected void usePIDOutput(double output) {
         if (topLimitSwitch.get() == true && output > 0) {
-            setSetpoint(LiftHeight.kMaxHeight.get());
+            setSetpoint(DesiredLiftHeight.kMaxHeight.get());
             liftDrive.stopMotor();
             return;
         }
         if (bottomLimitSwitch.get() == true && output < 0) {
             liftEncoder.reset();
```

Comment:
If we aren't using the encoder, we don't need to reset here. But maybe there is something we want to do with the lidar?